<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymptomPickerDialog</class>
 <widget class="QDialog" name="SymptomPickerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор симптома</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SearchWidget" name="m_searchWidget" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="m_stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="m_treeWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="m_tableView">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QWidget</extends>
   <header>../components/searchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_treeWidget</tabstop>
  <tabstop>m_buttonBox</tabstop>
  <tabstop>m_tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
